.gitignore

```diff
@@ -167,11 +167,12 @@ spine-cocos2dx/example/proj.android/app/.cxx
 spine-cocos2dx/example/build-win
 spine-godot/godot
 spine-godot/spine_godot/*.o
-spine-godot/spine_godot/spine-cpp
```

Comment:
Please do not remove this from the .gitignore file. The `spine-godot/build/setup.sh` script will copy `spine-cpp/` over to the appropriate place in the `spine-godot` directory.

Reply:
Will do, I'll go through the PR and clean a bunch of the other commits up or (remove completely).